src/index.test.js

```diff
@@ -0,0 +1,5 @@
+describe('To test JEST, will be removed', () => {
```

Comment:
While I do enjoy the tests, I would prefer this to be an actual test describing what we are testing:

describe "Jest functionality"
test "passes a basic truthy test"